cmake/find/Findtcl.cmake

```diff
@@ -0,0 +1,41 @@
+find_path(
+    TCL_INCLUDE_DIR
+    tcl.h
+    PATHS
+      "${TCL_ROOT}/include"
+    NO_DEFAULT_PATH
+)
+
+find_library(
+    TCL_LIBRARY
+    tcl8.6
+    PATHS
+      "${TCL_ROOT}/lib"
+    NO_DEFAULT_PATH
+)
+
+set(interface_libraries m z)
+if(ANDROID)
+  list(APPEND interface_libraries dl)
+elseif(APPLE)
+  list(APPEND interface_libraries pthread)
+else()
+  list(APPEND interface_libraries dl pthread)
+endif()
+
+if(NOT TARGET "tcl")
+  if(NOT TCL_LIBRARY)
+    message(FATAL_ERROR "TCL_LIBRARY not found!")
+  endif()
+  add_library("tcl" UNKNOWN IMPORTED)
```

Comment:
For imported libraries it's better to use `tcl::tcl`.

Reply:
fixed